src/relay/qnn/op/batch_matmul.cc

```diff
@@ -84,7 +84,9 @@ Expr MakeQuantizedBatchMatmul(Expr x, Expr y, Expr x_zero_point, Expr y_zero_poi
 
 Expr BatchMatmulFirstTerm(const Expr& quantized_x, const Expr& quantized_y,
                           const BatchMatmulAttrs* attrs) {
-  return MakeBatchMatmul(quantized_x, quantized_y, attrs->out_dtype);
+  ICHECK(attrs->transpose_a == false && attrs->transpose_b == true)
+      << "Currently qnn.batch_matmul only support NT format.";
```

Comment:
```suggestion
      << "Currently qnn.batch_matmul only supports (transpose_a=false, transpose_b=true).";
```

Reply:
I believe for enabling of transposed tensors for qnn.batch_matmul, it is required to set correct axes in `BatchMatmulSecondTerm` and `BatchMatmulThirdTerm` depending on attrs->transpose_a and attrs->transpose_b correspondingly